test/tools/events/ImportDescriptor.java

```diff
@@ -0,0 +1,27 @@
+package tools.events;
+
+import com.amazon.ion.SymbolTable;
+
+public class ImportDescriptor {
+    private final String importName;
+    private final int maxId;
+    private final int version;
+
+    public ImportDescriptor(SymbolTable symbolTable) {
+        this.importName = symbolTable.getName();
+        this.maxId = symbolTable.getMaxId();
+        this.version = symbolTable.getVersion();
+    }
+
+    final int getVersion() {
+        return version;
+    }
+
+    final int getMax_id() {
```

Comment:
The method names should also be camel case (no underscores).

Reply:
Changed